cli/src/template.pest

```diff
@@ -88,11 +91,18 @@ formal_parameters = {
   identifier ~ (whitespace* ~ "," ~ whitespace* ~ identifier)* ~ (whitespace* ~ ",")?
   | ""
 }
+string_pattern_identifier = @{ (ASCII_ALPHA | "_") ~ (ASCII_ALPHANUMERIC | "_" | "-")* }
```

Comment:
Just a comment. It's nice that you noticed that `-` is allowed as a pattern prefix whereas `-` can also be infix/prefix operators. The grammar isn't ambiguous yet, but it's a bit odd. I don't have a good idea to address this potential problem.